_topics/en/op_checktemplateverify.md

```diff
@@ -16,7 +16,7 @@ topic-categories:
 excerpt: >
   **OP_CHECKTEMPLATEVERIFY (CTV)** is a proposed new opcode that takes a
   commitment hash as a parameter and requires any transaction executing
-  the opcode include a set of outputs that match the commitment.  This
+  the opcode to include a set of outputs that match the commitment.  This
```

Comment:
probably better:

constrains the transaction to have an exact set of

nVersion
nLockTime
scriptSig hash (if any are non-null, otherwise restricted to all be null)
input count
sequences hash
output count
outputs hash
input index